WooCommerce/Classes/ViewRelated/Editor/AztecEditorView.swift

```diff
@@ -0,0 +1,26 @@
+import SwiftUI
+
+/// `UIViewControllerRepresentable` to use an Aztec Editor in SwiftUI.
+/// It can be used as-is, or be wrapped inside a parent View that manages its states and toolbar as needed.
+/// Params:
+/// - initialValue: Initial value in the editor
+/// - onContentChanged: callback to to be called when the content is changed.
+///
+struct AztecEditorView: UIViewControllerRepresentable {
+    let initialValue: String
+    let onContentChanged: (String) -> Void
+
+    func makeUIViewController(context: Context) -> AztecEditorViewController {
+        let controller = EditorFactory().customFieldRichTextEditor(initialValue: initialValue)
+
+        guard let aztecController = controller as? AztecEditorViewController else {
+            fatalError("EditorFactory must return an AztecEditorViewController, but returned \(type(of: controller))")
+        }
+
+        aztecController.onContentChanged = onContentChanged
+        return controller as! AztecEditorViewController
```

Comment:
Since you already use the guard check above, can you return `aztecController` here instead of `controller` with forced unwrapping?

Reply:
Oh huh this is definitely not intended to be there. I fixed this then I think it returned after one of the reverts and I forgot to update again.  Updated in https://github.com/woocommerce/woocommerce-ios/pull/13925/commits/adeb80deb0da453b447b1e01278ea50d00fe3c3e